__tests__/commands/install/integration.js

```diff
@@ -459,17 +459,30 @@ test.concurrent('install file: link file dependencies', async (): Promise<void>
 });
 
 test.concurrent('install file: protocol', (): Promise<void> => {
-  return runInstall({noLockfile: true}, 'install-file', async config => {
+  return runInstall({lockfile: false}, 'install-file', async config => {
     expect(await fs.readFile(path.join(config.cwd, 'node_modules', 'foo', 'index.js'))).toEqual('foobar;\n');
   });
 });
 
 test.concurrent('install with file: protocol as default', (): Promise<void> => {
-  return runInstall({noLockfile: true}, 'install-file-as-default', async config => {
+  return runInstall({}, 'install-file-as-default', async config => {
     expect(await fs.readFile(path.join(config.cwd, 'node_modules', 'foo', 'index.js'))).toEqual('foobar;\n');
   });
 });
 
+test.concurrent("don't install with file: protocol as default if target is a file", (): Promise<void> => {
+  // $FlowFixMe
+  return expect(runInstall({lockfile: false}, 'install-file-as-default-no-file')).rejects.toBeDefined();
```

Comment:
It is because of that. This was introduced by Jest 20 and @onurtemizkan upgraded flow type definitions and used it in one of his patches.

I think Flow gets confused by how `runInstall` is defined by `bind` and cannot determine the output type of `expect` here.